UnityProject/Assets/Scripts/Items/PDA/PDALogic.cs

```diff
@@ -12,11 +12,12 @@ namespace Items.PDA
 	[RequireComponent(typeof(ItemLightControl))]
 	[RequireComponent(typeof(ItemAttributesV2))]
 	[RequireComponent(typeof(PDANotesNetworkHandler))]
-	public class PDALogic : NetworkBehaviour, ICheckedInteractable<HandApply>, ICheckedInteractable<InventoryApply>
+	public class PDALogic : NetworkBehaviour, ICheckedInteractable<HandApply>, ICheckedInteractable<InventoryApply>, IServerInventoryMove
 	{
-		private const bool DEBUG_UPLINK = false;
-		private const string DEBUG_UPLINK_CODE = "Whiskey Tango Foxtrot-1337";
-		private const int TIME_BEFORE_UPLINK_ACTIVATION = 10;
+		// TODO: consider moving uplink code into its own class (perhaps compatible with pen, headset uplinks)
+
+		private readonly bool DEBUG_UPLINK = false;
```

Comment:
I feel like these variables would work better as a serializable field. Any reason to have the only be changed in the code itself?

Reply:
Done.